File: 1036.c
#include <stdio.h>
#include <math.h>

int main(){
    float a = 0, b = 0, c = 0, delta = 0, raiz= 0, r1 = 0, r2 = 0;
    scanf("%f %f %f", &a, &b, &c);
    delta = (b*b)-(4.0*a*c);
    if (a!=0 && delta>=0) {
        raiz = sqrt(delta);
        r1 = (-b+raiz)/(2.0*a);
        r2 = (-b-raiz)/(2.0*a);
        printf("R1 = %.5f\n", r1);
        printf("R2 = %.5f\n", r2);
    }
    else {
        printf("Impossivel calcular\n");
    }

    return 0;
}

File: 1036.js
// código está correto, uri ta bugado 

var input = require('fs').readFileSync('/dev/stdin', 'utf8');

var numbers = input.split(" ");
var A = parseFloat(numbers.shift()).toFixed(1);
var B = parseFloat(numbers.shift()).toFixed(1);
var C = parseFloat(numbers.shift()).toFixed(1);

var delta = parseFloat((B**2)-(4*A*C)).toFixed(1);

if(delta>0 && A!=0){
    var r1 = (-B + Math.sqrt(delta))/2*A;
    var r2 = (-B - Math.sqrt(delta))/2*A;
    
    console.log(`R1 = ${(r1/100).toFixed(5)}`);
    console.log(`R2 = ${(r2/100).toFixed(5)}`);
}

else{
    console.log("Impossivel calcular");
}